doc/changelog.md

```diff
@@ -2,6 +2,30 @@
 
 <h3>New features</h3>
 
+* Debug your compiled programs with Catalyst's new ``debug.print`` statement.
+  [(#279)](https://github.com/PennyLaneAI/catalyst/pull/279)
```

Comment:
```suggestion
* Debug compiled programs and print dynamic values at runtime with ``debug.print``
  [(#279)](https://github.com/PennyLaneAI/catalyst/pull/279)
```

Reply:
More descriptive? I also try to avoid saying 'new' in changelogs as it feels like editorializing, but that is a minor comment 😆

Reply:
Sounds good :) 

> I also try to avoid saying 'new' in changelogs as it feels like editorializing, but that is a minor comment 😆

Speaking of, I saw a bunch of 'now' when reviewing the changelog 😅

Reply:
oops 😬 I might be a hypocrite, 'now' sounds okay to me lol